### PR TITLE
Added issue attribute "is_private" 

### PR DESCRIPTION
**Pull request by fefurst**
According to #236

### CODE REVIEW

File: src/main/java/com/taskadapter/redmineapi/bean/Issue.java [hunk 1]
```diff
@@ -43,6 +43,7 @@
     private String statusName;
     private Version targetVersion;
     private IssueCategory category;
+    private Boolean isPrivate;
```
Comment (alexeyOnGitHub) [inline src/main/java/com/taskadapter/redmineapi/bean/Issue.java]: how about calling it "privateIssue" instead? having "is" prefix goes against Java Beans conventions

File: src/main/java/com/taskadapter/redmineapi/internal/RedmineJSONBuilder.java [hunk 2]
```diff
@@ -33,180 +33,179 @@
 public class RedmineJSONBuilder {
 
     /**
```
Comment (alexeyOnGitHub) [inline src/main/java/com/taskadapter/redmineapi/internal/RedmineJSONBuilder.java]: cannot review this part - too many changes in RedmineJSONBuilder. what happened to this class? why are hundreds of lines deleted and hundreds added? bad reformatting or moved methods?

**Comment by alexeyOnGitHub** [comment]
I appreciate your support :)
can you please address the review comments so that I can review again and merge?

**Comment by fefurst** [comment]
Sure ....

---------------------------------------
Felipe Furst

**Comment by brunojcm** [comment]
I agree that it privateIssue is better than isPrivate, mainly because the getter method would be isIsPrivate(), which is weird.

**Comment by fefurst** [comment]
Hi ...
Got it guys ... as soon as possible I'll change that ...

---------------------------------------
Felipe Furst

**Comment by alexeyOnGitHub** [comment]
this cannot be merged as-is.
I added support for parsing this field  in https://github.com/taskadapter/redmine-java-api/commit/82b8096cacb8b3c3a01d59144d49e15a57361b2c

**Pull request closed without merging**